content/api/recipient-lists.apib

```diff
@@ -584,21 +550,20 @@ number of rejected recipients will only be returned if a `recipients` array is p
             }
 
 
-## Delete [/recipient-lists/{id}]
+### Delete a Recipient List [DELETE /recipient-lists/{id}]
+
 
-### Delete a Recipient List [DELETE]
 
 Permanently delete the specified recipient list.
 
++ Parameter
+    + id (required, string, `unique_id_4_graduate_students_list`)
+
 Once a recipient list is deleted, it
-cannot be recovered.  Before deleting a list, ensure that it is no longer needed and keep a backup copy.  If a deleted
-list is needed again, the list must be resubmitted with the CREATE API.
+cannot be recovered.  Before deleting a list, ensure that it is no longer needed and keep a backup copy.
 
 <banner status="warning">If a transmission contains a recipient list, the recipient list cannot be deleted if the transmission is submitted or generating.</banner>
```

Comment:
this also isn't rendering for me

Reply:
Had it in the wrong order cuz of unrelated testing :headdesk: